tableauserverclient/server/request_options.py

```diff
@@ -62,8 +62,9 @@ def __init__(self, pagenumber=1, pagesize=None):
         self.pagesize = pagesize or config.PAGE_SIZE
         self.sort = set()
         self.filter = set()
+        self.fields = set()
         # This is private until we expand all of our parsers to handle the extra fields
-        self._all_fields = False
```

Comment:
I would guess people have used this field even though it's private, no? Can we support both for a release?